ItsME/Presentation/Scenes/TotalCV/TotalCVViewController.swift

```diff
@@ -217,36 +236,58 @@ private extension TotalCVViewController {
 extension TotalCVViewController: UITableViewDelegate, UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return viewModel.resumeCategory[section].items.count
+        
+        if tableView == categoryTableView {
+            return viewModel.resumeCategory[section].items.count
+        } else {
+            return 1
+        }
     }
     
     func numberOfSections(in tableView: UITableView) -> Int {
-        return viewModel.resumeCategory.count
+        
+        if tableView == categoryTableView {
+            return viewModel.resumeCategory.count
+        } else {
+            return viewModel.coverLetterItems.count
+        }
     }
     
     func tableView(_ tableView: UITableView,
                    viewForHeaderInSection section: Int) -> UIView? {
-        let view = tableView.dequeueReusableHeaderFooterView(withIdentifier: CategoryHeaderView.reuseIdentifier) as? CategoryHeaderView
         
-        view?.titleLabel.text = viewModel.resumeCategory[section].title
-        
-        return view
+        if tableView == categoryTableView {
+            let categoryView = tableView.dequeueReusableHeaderFooterView(withIdentifier: CategoryHeaderView.reuseIdentifier) as? CategoryHeaderView
+            categoryView?.titleLabel.text = viewModel.resumeCategory[section].title
+            
+            return categoryView
+        } else {
+            let cvView = tableView.dequeueReusableHeaderFooterView(withIdentifier:CoverLettersHeaderView.reuseIdentifier) as? CoverLettersHeaderView
```

Comment:
```suggestion
            let cvView = tableView.dequeueReusableHeaderFooterView(withIdentifier: CoverLettersHeaderView.reuseIdentifier) as? CoverLettersHeaderView
```
코드 컨벤션에 어울리도록 띄어쓰기 한 칸 추가했습니다 :)